ARCHITECTURE.md

```diff
@@ -76,7 +76,7 @@ Outside of the core C++ static library, platform SDKs typically rely on platform
 
 * For iOS and macOS this means Xcode and the xcodebuild command line tool.
 * For Android, Gradle and Android Studio.
-* For Qt, `qmake`.
+* For Qt, plain CMake is used to generate the build system.
```

Comment:
Maybe the wording is not ideal, CMake just generates the makefiles here.

Reply:
In that case the Makefiles are the build system, right?

Reply:
@ntadej Could you suggest a different wording?

Reply:
Let's just keep it as is.